docs-requirements.txt

```diff
@@ -37,3 +37,6 @@ sqlalchemy>=1.0
 tornado>=5.1.1
 ddtrace>=0.34.0
 httpx>=0.18.0
+
+# pin to compatible versions
+itsdangerous==2.0.1
```

Comment:
What is `itsdangerous`? Is this used anywhere?

Reply:
It is an indirect dependency that needs to be pinned as newer versions have breaking changes. I don't remember which package it is a direct dependency of.